### PR TITLE
[parse-decimal-number] Add types

### PR DESCRIPTION
**Pull request by ciaran1344**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/parse-decimal-number/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    /**
+     * Thousands separator character.
+     * Default: ",".
+     */
+    thousands: string;
+}
+
+type Options = ArrayOptions | HashOptions | string;
+
+declare const parseDecimalNumber: {
```
Comment (jakebailey) [inline types/parse-decimal-number/index.d.ts:23]: This seems like a surprising way to declare this API; why not the more typical namespace + function declaration?

**Comment by AlexJerabek** [comment]
Why are @OfficeDev and several Office folks tagged on this PR?

**Comment by jakebailey** [comment]
I have no idea; it randomly applied some tags to this PR that it shouldn't have. I don't know if it got confused with #63745 or what.

**Review by jakebailey — APPROVED** [review]

**Comment by ciaran1344** [comment]
Ready to merge

**Comment by webstech** [comment]
> I have no idea; it randomly applied some tags to this PR that it shouldn't have. I don't know if it got confused with #63745 or what.

It looks like three recently completed PRs were merged onto the branch used in this PR.  The code owners of those types got included.  Possibly a 🪲?

**Comment by ciaran1344** [comment]
I did `git pull --rebase upstream master` before pushing my second commit, so maybe those interim commit owners got added? Never seen that behavior before, so likely a bug.

**Pull request merged**